src/us/kbase/workspace/test/kbase/HandleAndShockTest.java

```diff
@@ -506,32 +494,23 @@ public void basicHandleTest() throws Exception {
 				.withObjid(2L), new ObjectIdentity().withWorkspace(workspace)
 				.withObjid(1L)))).get(0);
 		checkExternalIDError(ret.getHandleError(), ret.getHandleStacktrace());
-		
-		checkReadAcl(node, twouser);
-		
+
+		checkReadAcl(node, oneuser);
+
 		//test error message for deleted node
 		node.delete();
-		
+
 		ObjectData wod = CLIENT2.getObjects2(new GetObjects2Params()
 				.withObjects(Arrays.asList(new ObjectSpecification()
 					.withWorkspace(workspace)
 					.withObjid(1L)))).getData().get(0);
-		
+
 		@SuppressWarnings("unchecked")
 		Map<String, Object> retdata = wod.getData().asClassInstance(Map.class);
 		assertThat("got correct data", retdata, is(handleobj));
-		
```

Comment:
These errors are expected by the test. Why did this change? It seems like many tests have been removed.

Reply:
👍